BKG/Booking.API/DependencyInjection.cs

```diff
@@ -11,6 +14,8 @@ public static void AddDataAccessDependencies(this IServiceCollection services, I
             services.AddDbContext<BookingDbContext>(opt =>
```

Comment:
You can register your dbContext in the DAL's layer extension method and pass IConfiguration through DI into this method. Hope to see it in future pull request)

Reply:
You're right, but this is the next task, so you can see changes in the next pull request